Formula/kim-api.rb

```diff
@@ -21,11 +22,18 @@ class KimApi < Formula
   depends_on "doxygen" => :build
   depends_on "gcc" # for gfortran
```

Comment:
```suggestion
  depends_on "gcc"
```
You're now using `gcc` for the `g{cc,++}` compiler as well, so the comment needs to be removed.